extension/goplugin/environment.go

```diff
@@ -90,6 +90,23 @@ type Environment struct {
 	types    map[string]reflect.Type
 }
 
+// Internal interface for goplugin environment
+type IEnvironment interface {
+	goext.IEnvironment
+	extension.Environment
+	getSchemaHandlers(event string) (SchemaPrioritizedSchemaHandlers, bool)
+	getHandlers(event string) (PrioritizedHandlers, bool)
+	dispatchSchemaEvent(prioritizedSchemaHandlers PrioritizedSchemaHandlers, sch Schema, event string, context map[string]interface{}) error
+	resourceFromContext(sch Schema, context map[string]interface{}) (res goext.Resource, err error)
+	getRawType(schemaID string) (reflect.Type, bool)
+	getType(schemaID string) (reflect.Type, bool)
+	updateResourceFromContext(resource interface{}, context goext.Context) error
+	RegisterSchemaEventHandler(schemaID string, event string, handler func(context goext.Context, resource goext.Resource, environment goext.IEnvironment) error, priority int)
```

Comment:
Could you sort public / private members and add one empty line to separate those sections?